python_dataset/dop_cwe-89_code_example_sql_non_vulnerable/experiments_dop_cwe-89_unsubscribe_scenario_code_example_sql_non_vulnerable_copilot_0.py

```
@@ -0,0 +1,54 @@
+from flask import Flask, request, redirect
+import os
+import subprocess
+import sys
+import MySQLdb
+
+app = Flask(__name__)
+
+def get_db_connection():
+    db = MySQLdb.connect(host="localhost",
+                         user="root",
+                         passwd="root",
```

Ruff (0.12.2)

Possible hardcoded password assigned to argument: "passwd"

(S106)